charts/grafana/FLUX.md

```diff
@@ -1,12 +1,29 @@
 # Flux language setup
-### Manual Setup
-* Flux is still in beta, so this information may be out of date
-* Follow instructions laid out by [Flux Datasource for Grafana]
-(https://grafana.com/plugins/grafana-influxdb-flux-datasource)
+## Manual Setup
+- [Flux](https://docs.influxdata.com/flux/v0.12/) is still in beta, so this information may be out of date
+- [Enable](https://docs.influxdata.com/flux/v0.12/introduction/installation) Flux by setting the `flux-enabled` option to `true` under the `http` section of the influxdb.conf
+- Follow instructions laid out by [Flux Datasource for Grafana](https://grafana.com/plugins/grafana-influxdb-flux-datasource)
 
+## Cronus-Monitoring Setup
+- Run the init.sh script in the scripts directory and the included helm chart will enable flux automatically
+    - Flux is enabled in Grafana via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/grafana/values.yaml) `datasources` with `grafana-influxdb-flux-datasource` as the `type` 
+    
+    - Flux is enabled in InfluxDB via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/influxdb/values.yaml) `[http]` with `flux_enabled` set to `true` [influxdb/templates/config.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/influxdb/templates/config.yaml) takes that value and deploys the flux enabled InfluxDB
 
-### Helpful information
-* Read more about Flux [here](https://github.com/influxdata/platform/tree/master/query)
-* Use the limit function to reduce the number of datapoints so that the master node of the kubernetes cluster doesn't get dropped
+## Helpful information
+- Read more about [Flux](https://github.com/influxdata/platform/tree/master/query)
 
+## How to begin writing queries in grafana
+1. Open the dashboard intended, then create a panel and open that panel's editor
+2. Click the drop down menu for `Data Sources`, then click the `influx-flux` data source for the current panel
+3. The query editor will switch to a flux IDE
+    > Note: an enter key press will cause the current query to run. Use shift+enter to produce a return character. Additionally the Grafana IDE autocompletion is very aggressive with a press of the tab key autocompleting several lines of a query at a time
```

Comment:
is clunky
"Note: Pressing the Enter key will ..."

Reply:
Also the possessive form, "IDE's"

Reply:
also add a comma to split that last idea in the note

Reply:
I think the way I updated it in my latest commit should have made it better? I used the more active phrasing you took, made IDE possessive and added the comma